src/main/java/kernitus/plugin/OldCombatMechanics/OCMMain.java

```diff
@@ -3,6 +3,7 @@
 import kernitus.plugin.OldCombatMechanics.hooks.PlaceholderAPIHook;
 import kernitus.plugin.OldCombatMechanics.hooks.api.Hook;
 import kernitus.plugin.OldCombatMechanics.module.*;
+import kernitus.plugin.OldCombatMechanics.module.Module;
```

Comment:
why is this import required? we are importing `.module.*` on the line above already :thinking:

Reply:
this should be removed

Reply:
It was conflicting with java.util.Module, i had to compile it under jdk 11 so my ide was complaining